<?xml version="1.0"?>
<launch>
	<group ns="claw">
		<node name="claw_server_2024" pkg="behaviors" type="2024_claw_server.py" output="screen"/>
		<rosparam command="load" file="$(find behaviors)/config/2024_claw_config.yaml"/>
	</group>
</launch>